.github/workflows/end-to-end-test-portals.yml

```diff
@@ -0,0 +1,90 @@
+name: End to end testing for Portals
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+env:
+  PW_ALL_BLOBS_DIR: all-blob-reports
+  PORTALS_SUBDIR: apps/portals
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  e2e-tests:
+    if: ${{ (github.event_name == 'push' && (github.ref_name == 'main' || github.actor == github.repository_owner)) || github.event_name == 'pull_request' }}
```

Comment:
Run e2e tests on pull request triggers and on push to main for the Sage repo and push to any branch for non-Sage repos